index.html

```diff
@@ -2,12 +2,65 @@
 <html>
 <head>
 <link href="style.css" rel="stylesheet" type="text/css">
+<script src="https://kit.fontawesome.com/173db1d6d6.js" crossorigin="anonymous"></script>
 <title>Portafolio</title>
 </head>
 <body>
 
-<h1>This is a Heading</h1>
-<p>This is a paragraph.</p>
+    <div class="wrapper">
 
+        <header class="toolbar">
+            <nav>            
+                <a href="#">
+                <div class="item">Edahí</div> </a>        
+                <i class="fa-solid fa-bars"></i>                    
+            </nav>
+        </header>
+        
+        <section>
+            <p class="info">Hey There. <br>
+                I’m Edahí</p>
+    
+            <p class="intro">I’am a Software Developer</p>   
+        
+            <p> I can help you build a product ,
+                feature or website Look through some of my work
+                and experience! If you like what you see and have
+                a project you need coded,  don’t hestiate to contact me. 
+            </p>    
+        </section>
+
+        <section>
+            <!-- Here future sections will be included --> 
+        </section>
+            
+    
+        <div class="social-group">
+            <ul>
+                <li class="social">
+                    <a href="http://github.com/edahigure">
+                        <i class="fa fa-github"></i></a>
+                </li>
+                <li class="social">
+                    <a href="https://linkedin.com/in/edahigure">
+                        <i class="fa-brands fa-linkedin-in"></i></a>
+                </li>
```

Comment:
- [Optional] Please use an indentation of two spaces ina ll your documents for readability. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

- [Optional] Please make sure that each of your commit messages should be written in the imperative tense and capitalized. This will make your work look more professional. A good example of a professional commit message is Add markup to index.html and a bad example is added HTML. [Read more about how to write descriptive commit messages](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8).